docs/test.md

```diff
@@ -60,11 +60,16 @@ Next you need to build a QA version of the Triton Docker image. This
 image will contain Triton, the QA tests, and all the dependencies
 needed to run the QA tests. First do a [Docker image
 build](build.md#building-triton-with-docker) to produce the
-*tritonserver_build* and *tritonserver* images. Then build the
-*tritonserver_sdk* image that contains the client libraries,
-model analyzer, and examples.
+*tritonserver_build* and *tritonserver* images.
+
+Then build the *tritonserver_sdk* image that contains the client
+libraries, model analyzer, and examples using the following
+commands. You must first checkout the <client branch> branch of the
+*client* repo into the clientrepo/ subdirectory. Typically you want to
+set <client branch> to be the same as your current server branch.
 
 ```
+$ git clone --single-branch --depth=1 -b <client branch> https://github.com/triton-inference-server/client.git clientrepo
```

Comment:
We should add a cd command after the git clone

Reply:
You don't cd into the client repo... it just needs to be present